tests/compatibility-test.py

```diff
@@ -198,6 +193,7 @@ def test_ray_serve(self):
         exit_code, _ = utils.exec_run_container(container, f'python3 /usr/local/test_ray_serve_1.py {ray_namespace}', timeout_sec = 180)
 
         if exit_code != 0:
+            show_cluster_info("default")
```

Comment:
I found that we hard-coded a lot of "default" in both `test_ray_serve` and `test_detached_actor`. How about creating a local variable in each test to replace this hard-coded method?

For example,

`utils.get_pod(namespace='default', label_selector='ray.io/node-type=head')`

Reply:
I have updated in https://github.com/ray-project/kuberay/pull/775/commits/d5c8fd6eb15614893f700495bf03bd7a4ff26a33 and https://github.com/ray-project/kuberay/pull/775/commits/c0d8ef65846495b62375e3783239e9ded2ea8722. To be noted, these two functions will utilize the same raycluster which is created in ```setUpClass()```. If we want to separate the namespace in the future, we will need to move the creation of raycluster into corresponding functions.